CHANGELOG.md

```diff
@@ -4,6 +4,9 @@ All notable changes to this project will be documented in this file.
 
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+### v0.17.6
```

Comment:
Hi @Paulokim1 - thanks for this. As per https://github.com/data-8/datascience/issues/559, can we please increment the package version to 0.18.0? Thanks!